cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/GoalOptimizer.java

```diff
@@ -549,7 +549,7 @@ public void run() {
         return;
       }
       OperationProgress operationProgress =
-          _progressUpdateLock.compareAndSet(false, true) ? _proposalPrecomputingProgress : new OperationProgress();
+          _progressUpdateLock.compareAndSet(false, true) ? _proposalPrecomputingProgress : new OperationProgress("");
```

Comment:
didn't you explicitly overload OpeartionProgress constructor so that no arguments will still work?